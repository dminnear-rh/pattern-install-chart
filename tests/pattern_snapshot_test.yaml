suite: Test pattern snapshot
templates:
  - templates/pattern.yaml
release:
  name: release-test
tests:
  - it: gitspec.targetRepo should match snapshot
    asserts:
      - matchSnapshot:
          path: spec.gitSpec.targetRepo
