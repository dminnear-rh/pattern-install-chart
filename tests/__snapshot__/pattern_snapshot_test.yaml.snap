gitspec.targetRepo should match snapshot:
  1: |
    https://github.com/pattern-clone/mypattern
